File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Kiểm tra số chính phương/SoCPhuong.cs
using System;
namespace SoCP
{
    public class Program
    {
        static void Main(string[]args)
        {
            int a = int.Parse(Console.ReadLine());
            string temp = "";
            for(int i = 0; i < a; i++)
                if(i * i == a)
                    temp = "true";
            if(temp == "true")
                Console.Write("yes");
            else
                Console.Write("no");
        }
    }
}

File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Kiểm tra số chính phương/SoCPhuong.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int a;
    cin >> a;
    string temp = "";
    for(int i = 0; i < a; i++)
        if(i * i == a)
            temp = "true";
    if(temp == "true")
        cout << "yes";
    else
        cout << "no";
}